web/views/shared/head.erb

```diff
@@ -0,0 +1,3 @@
+<% add_to_head do %>
+  <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" media="screen" rel="stylesheet" type="text/css" />
```

Comment:
I think it is an overkill to bring boostrap. SidekiqWeb is meant to be very small app with as little external dependencies as possible. Also, I think it is not a good idea to rely on external CDN from within a library. Many Sidekiq run inside enterprise environments with restricted internet access.

Having said all that,  can I ask to to remove bootstrap dependency, come up with custom CSS classes and bundle them as SidekiqWeb's assets?